pages/common/xml.md

```diff
@@ -0,0 +1,29 @@
+# xml
+
+> XMLStarlet Toolkit: Query, edit, check, convert and transform XML documents.
+> Subcommands: `canonic`, `edit`, `elements`, `escape`, `unescape`, `format`, `list`, `pyx`, `depyx`, `select`, `transform`, or `validate`.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- Execute a subcommand with input from a file or URI and using standard output:
```

Comment:
Yes, I agree. However, some of your reviewers disagree with others on this an other points. I have been asked to spell things out, abbreviate, remove words for brevity, add words (like "the") to "read better". Please put this stuff into the guidelines so that authors and reviewers are all working from the same guidelines. This reminds me too much of customers that change their mind with every new voice on their team.

❯ grep "standard output" *.md | wc -l
      31
❯ grep "standard input" *.md | wc -l
      33
❯ grep "stdout" *.md | wc -l
      89
>grep "stdin" *.md | wc -l
      78

stdin/stdout wins be more than 2 to 1.

How 'bout we revise all the ones that say "standard input" and "standard output"?

Reply:
xml.md:- Execute a subcommand with input from a file or URI, printing to stdout:
xml.md:- Execute a subcommand using stdin and stdout:
xml-depyx.md:- Convert a PYX document from stdin to XML format:
xml-escape.md:- Escape special XML characters from stdin:
xml-format.md:- Format an XML document from stdin, removing the DOCTYPE declaration:
xml-pyx.md:- Convert an XML document from stdin to PYX format:
xml-unescape.md:- Unescape special XML characters from stdin: